src/software/ai/hl/stp/tactic/shadow_freekicker_tactic.cpp

```diff
@@ -76,7 +76,7 @@ void ShadowFreekickerTactic::calculateNextAction(ActionCoroutine::push_type &yie
 
         move_action->updateControlParams(
             *robot, defend_position, (ball.position() - robot->position()).orientation(),
-            0, DribblerMode::OFF, AutochickType::NONE, BallCollisionType::AVOID);
+            0, DribblerMode::OFF, BallCollisionType::AVOID);
```

Comment:
is there a reason we dont use `autochip_move_action` in `shadow_freekicker_tactic` like we do in `shdow_enemy_tactic` ?

Reply:
Since we don't autochip in this Tactic, we can just use the simpler MoveAction